src/helpers/deepLinking.ts

```diff
@@ -20,21 +20,35 @@ const urlSafeBase64Decode = (data: string) => {
 };
 
 export const getInviteLink = (contact: InvitedContact, profileName: string): string => {
+    return `${BASE_URL}${INVITE}${makeInviteParams(contact, profileName)}`;
+};
+
+export const getInviteLinkWithParams = (params: string): string => {
+    return `${BASE_URL}${INVITE}${params}`;
+};
+
+const makeInviteParams = (contact: InvitedContact, profileName: string): string => {
```

Comment:
@agazso tbh it doesn't feel right to have the parameters with different encoding (base64, raw string, urlEncoded), wdyt?

Reply:
I don't think it's important, because the links are meant to be copy-pasted and not for human input.

However it's more important that the links should be short, because the longer they are the harder to scan the QR codes that are generated from them, especially under not ideal light conditions. Also long links in messenger apps look messy and are hard to use, so basically it's a UX issue.

That was the original reason that the `randomSeedBytes` and `contactPublicKeys` were base64 encoded, because they contain 32 bytes binary data. If we used hexstrings they would have been 64 bytes each but with base64 encoding they can be stored in 44 bytes.